src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+

```

Comment:
mate checkstyle doesn't provide for a line after class definition, please fix everywhere
```suggestion
```